web-client/src/views/AdvancedSearch/PractitionerSearchResults.tsx

```diff
@@ -25,23 +24,21 @@ export const PractitionerSearchResults = connect(
           <>
             <h1 className="margin-top-4">Search Results</h1>
             <div ref={paginatorTop}>
-              {practitionerSearchHelper.numberOfResults &&
-                practitionerSearchHelper.numberOfResults >
-                  PRACTITIONER_SEARCH_PAGE_SIZE && (
-                  <Paginator
-                    breakClassName="hide"
-                    forcePage={practitionerSearchHelper.activePage || 0}
-                    marginPagesDisplayed={0}
-                    pageCount={practitionerSearchHelper.pageCount || 0}
-                    pageRangeDisplayed={0}
-                    onPageChange={pageChange => {
-                      submitPractitionerNameSearchSequence({
-                        selectedPage: pageChange.selected,
-                      });
-                      focusPaginatorTop(paginatorTop);
-                    }}
-                  />
-                )}
+              {practitionerSearchHelper.showPaginator && (
+                <Paginator
+                  breakClassName="hide"
+                  forcePage={practitionerSearchHelper.activePage}
```

Comment:
I believe these should be defaulted to 0 in the helper.

Reply:
it's being defaulted in the helper in the block starting on line 45, except `activePage`, which is defaulted in `defaultAdvancedSearchFormAction` because it's shared across a few components. would it make sense to default it another way?

Reply:
Hmmm I only see it referenced in the helper and the tsx, are there other places I am missing? To me, I don't see a reason it needs to be a value on state.

Reply:
makes sense to me!

Reply:
I feel I should do *some* explaining:
You were right that `activePage` doesn't need to be on state, so I removed it. but when it changes, it reads the value from `pageNum` in state, which is set when the "search" button or any of the paginator buttons is clicked. The "search" button is on a different component than the paginator is on.